client.go

```diff
@@ -221,3 +229,22 @@ type TemplatePreview struct {
 	Subject string `json:"subject,omitempty"`
 	Body    string `json:"body"`
 }
+
+type Notification struct {
+	ID           string `json:"id,omitempty"`
+	Subject      string `json:"subject"`
+	Body         string `json:"body"`
+	Reference    string `json:"reference"`
+	EmailAddress string `json:"email_address"`
+	PhoneNumber  string `json:"phone_number"`
+	Type         string `json:"type"`
+	Status       string `json:"status"`
+	CreatedBy    string `json:"created_by_name"`
```

Comment:
@andrewoh531 `created_by_name`?

Reply:
We don't have any API around users. It would be useless to return a userID -- I assume that's why this gives back a name

Reply:
I searched for `created_by_name` in the codebase and got 0 results. Did I do a bad search?

Reply:
It's in the client-sdk for python, node

Reply:
https://github.com/govau/notify-client-dotnet/blob/1ef5ff29fd52836b3896d24b11ef8f3a1a943d9d/src/Notify/Models/Notification.cs#L38

Reply:
We worked it out offline. We haven't pulled in this field from upstream yet.